### PR TITLE
additonal changes + plugins

### PR DESCRIPTION
**Pull request by Kyleeld**
additonal changes + plugins

raidstheiving
lootingbagviewer
stronghold
warindicators
rememberclan

additional tags set.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/lootingbagviewer/LootingBagViewerPlugin.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import net.runelite.client.plugins.PluginDescriptor;
+import net.runelite.client.ui.overlay.OverlayManager;
+
+import javax.inject.Inject;
+
+@PluginDescriptor(
+	name = "PvP Looting Bag Viewer",
+	description = "Add an overlay showing the contents of your looting bag",
+	tags = {"alternate", "items", "overlay", "second"},
+	enabledByDefault = false
```
Comment (zeruth) [inline runelite-client/src/main/java/net/runelite/client/plugins/lootingbagviewer/LootingBagViewerPlugin.java]:
```suggestion
enabledByDefault = false,
type = "PVP"
```

Comment (zeruth) [reply runelite-client/src/main/java/net/runelite/client/plugins/lootingbagviewer/LootingBagViewerPlugin.java]: After that, looks good to me :)

Comment (Kyleeld) [reply runelite-client/src/main/java/net/runelite/client/plugins/lootingbagviewer/LootingBagViewerPlugin.java]: tag Added

**Comment by Kyleeld** [comment]
deleted the following plugins:

safespot
group list
zoneindicators

**Pull request merged**